CMakeLists.txt

```diff
@@ -9,6 +9,7 @@ project(
     LANGUAGES CXX
 )
 
+include(GNUInstallDirs)
```

Comment:
I'm not sure why that was missing, but please make sure this line is the only one in exemplar/ which includes GNUInstallDirs

Reply:
Yeah I am surprised as well.

Confirmed that this is the only one in exemplar that includes GNUInstallDirs.